projects/components/src/table/table-api.ts

```diff
@@ -1,14 +1,18 @@
 import { Dictionary } from '@hypertrace/common';
 import { Observable } from 'rxjs';
 import { FilterAttribute } from '../filter-bar/filter-attribute';
+import { TableCellParserConstructor } from './cells/table-cell-parser';
+import { TableCellRendererConstructor } from './cells/table-cell-renderer';
 import { TableCellAlignmentType } from './cells/types/table-cell-alignment-type';
 
 export interface TableColumnConfig {
   field: string;
+  display?: string;
+  renderer?: TableCellRendererConstructor;
```

Comment:
In other places we seem to assume renderer and parser are defined. This feels like we're mixing up internal and external representations. We shouldn't require these in addition to display, since display encompasses both of them.